aot/src/main/java/com/dylibso/chicory/aot/AotMethods.java

```diff
@@ -183,6 +183,9 @@ public static Value[] callIndirect(
         }
 
         checkInterruption();
+        // TODO: verify
+        // here we should not pass through the external "call" method
+        // but directly emit the invocation of the underlying function
```

Comment:
Agreed, this could be improved, but we still need to put the lookup table somewhere. This should all be replaced with invoke dynamic.

Reply:
tracked in #540